support-frontend/cloud-formation/CODE-deploy.sh

```diff
@@ -5,4 +5,5 @@ aws cloudformation deploy \
     --stack-name support-CODE-frontend \
     --template-file cfn.yaml \
     --region eu-west-1 \
-    --capabilities CAPABILITY_IAM
\ No newline at end of file
+    --capabilities CAPABILITY_IAM \
+    --tags App=frontend Stage=CODE Stack=support
```

Comment:
this line was missing and was causing the CLI to remove the tags, which meant that riffraff couldn't find the code stack any more!

Reply:
That explains a lot!